### PR TITLE
Update spanish translation

### PR DESCRIPTION
**Pull request by likeawindrammer**

### CODE REVIEW

File: resource/translations/ao_es.ts [hunk 1]
```diff
@@ -37,7 +38,7 @@ Que tengas un buen día.</translation>
     <message>
         <location filename="../../src/packet_distribution.cpp" line="293"/>
         <source>Attorney Online 2</source>
-        <translation>Attorney Online 2</translation>
+        <translation></translation>
```
Comment (stonedDiscord) [inline resource/translations/ao_es.ts:41]: what

Comment (oldmud0) [reply resource/translations/ao_es.ts:41]: mission accomplished

Comment (likeawindrammer) [reply resource/translations/ao_es.ts:41]: The client will just use the original string. So no need to copy paste.

**Comment by likeawindrammer** [comment]
An updated version of this will be merged to 2.7-rc branch

**Pull request closed without merging**